Source/ChilliSource/Core/State/State.cpp

```diff
@@ -176,13 +198,13 @@ namespace ChilliSource
     //------------------------------------------
     Scene* State::GetScene()
```

Comment:
I wanted to rename it but didn't because it was a breaking API change. However it is a simple find and replace so will do that